cluster/gce/gci/configure-helper.sh

```diff
@@ -2713,13 +2713,24 @@ function main() {
 
   KUBE_HOME="/home/kubernetes"
   KUBE_BIN=${KUBE_HOME}/bin
-  PYTHON="python"
   CONTAINERIZED_MOUNTER_HOME="${KUBE_HOME}/containerized_mounter"
   PV_RECYCLER_OVERRIDE_TEMPLATE="${KUBE_HOME}/kube-manifests/kubernetes/pv-recycler-template.yaml"
 
-  if [[ "$(python -V)" =~ "Python 3" ]]; then
+  if [[ "$(python -V)" =~ "Python 2" ]]; then
+    # found python2, just use that
+    PYTHON="python"
+  elif [[ -d "/usr/bin/python2.7" ]]; then
+    # found python2 in a different directory
```

Comment:
# System python not defaulted to 2 but using 2.7 during migration

Reply:
Doing that now